code/modules/holiday/holidays.dm

```diff
@@ -402,14 +402,14 @@
 	begin_month = JUNE
 	begin_weekday = SUNDAY
 
-/datum/holiday/moth
-	name = "Moth Week"
+/datum/holiday/insect
```

Comment:
did this get like, caught up in a find and replace? The holiday is called moth week, no reason it needs to be changed.